starlite/utils/cli.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
```

Comment:
Yes, this whole module is a bit hacky but doing this enables us to provide a way to dynamically register commands, that doesn't break if the extra dependencies for the CLI are not installed. This could be particularly useful for external libraries that want to hook into the CLI (think `starlite users create ...`)